tests/datasource/test_new_datasource_with_runtime_data_connector_sql_execution_engine.py

```diff
@@ -108,92 +108,6 @@ def test_datasource_with_runtime_data_connector_and_sqlalchemy_execution_engine_
     assert available_data_asset_names == expected_available_data_asset_names
 
 
-def test_datasource_with_runtime_data_connector_and_sqlalchemy_execution_engine_unknown_datasource(
```

Comment:
@donaldheppner Same question about tests deletion.  I imaging that there must be one reason covering all such changes.  Thanks.

Reply:
These tests check the RuntimeBatchRequest constructor for the same logic that is tested other places and in test_batch.py -- the actual assertions here have nothing to do with SqlAlchemy.